src/Grid/grid.jl

```diff
@@ -898,3 +972,20 @@ function OrientationInfo(surface::NTuple{N, Int}) where N
     end
     return OrientationInfo(flipped, shift_index)
 end
+
+# Dispatches for facets
+
+@inline facets(c::AbstractCell{<:AbstractRefShape{1}}) = map(i -> (i,), vertices(c)) # facet always tuple of tuple
+@inline facets(c::AbstractCell{<:AbstractRefShape{2}}) = edges(c)
+@inline facets(c::AbstractCell{<:AbstractRefShape{3}}) = faces(c)
```

Comment:
should be moved directly under or above the `edge`,`face`,`vertices` definition of `AbstractCell` imho